tests/test_e2e.py

```diff
@@ -2,45 +2,36 @@
 from selenium import webdriver
 from dotenv import load_dotenv, find_dotenv
 from selenium.webdriver.common.keys import Keys
-from trello import TrelloClient
 import os
-import requests
+import pymongo
 import pytest
 import app
+import random
+import string
 
-TRELLO_URL = "https://api.trello.com/1"
+def get_db_collection():
 
-def create_trello_board():
-    response = requests.post(
-        url=f'{TRELLO_URL}/boards',
-        params={
-            'key': os.getenv("API_KEY"),
-            'token': os.getenv("TOKEN"),
-            'name': 'Selenium Test Board',
-        }
-    )
-    return response.json()['id']
+    dbClientUri = os.getenv('MONGO_DB_CONNECTION')
+    databaseName = os.getenv('MONGO_DB_DATABASE')
+    collectionName = 'collection'
 
-def delete_trello_board(board_id):
-    requests.delete(
-        url=f'{TRELLO_URL}/boards/{board_id}',
-        params={
-            'key': os.getenv("API_KEY"),
-            'token': os.getenv("TOKEN"),
-        }
-    )
+    dbClient = pymongo.MongoClient(dbClientUri)
+    db = dbClient[databaseName]
+    return db[collectionName]
     
 @pytest.fixture(scope='module')
 def test_app():
-    file_path = find_dotenv('.env')
+
+    file_path = find_dotenv('.env.test')
     load_dotenv(file_path, override=True)
```

Comment:
Since you've deleted `.env.test` this loads the real `.env` file. That's fine for the e2e tests, but the integration tests shouldn't have real credentials in case they start relying on them